### ISSUE TITLE
Broken logo image on hangout reminder email notification

### ISSUE THREAD

**Issue by lpatmo** [codebuddiesdotorg/cb-v2-scratch#206]
Screenshot of hangout reminder email notification: 
![screen shot 2016-09-20 at 11 46 24 pm](https://cloud.githubusercontent.com/assets/4512699/18700521/8d6944a8-7f8c-11e6-937b-9c164a5d1362.jpg)

**Comment by lcrossan** [comment]
Hey, I'd like to fix this. What do you need?

**Comment by William-R-Wilson** [comment]
this ought to get you started:  https://github.com/codebuddiesdotorg/cb-v2-scratch#how-do-i-contribute-to-this-project

**Comment by sergeant-q** [comment]
@lcrossan as Will noted, feel free to create a new fix branch and then submit a PR and we'll review it! Let me know if you have any questions in the meantime!

**Comment by lcrossan** [comment]
Sorry if I wasn't clear -- Git fundamentals are great, but I was actually asking about the issue itself! What would a fix look like? What image would y'all want in the email? Are there any pain points on this project in particular that I would likely run into on this?

Thanks!

**Comment by sergeant-q** [comment]
@lcrossan the logo in the email reminder should be the CB logo we are using throughout the site. The email reminder is using an HTML template that **references** the logo image under `public/images`. You can find the template in `private/email-hangout-reminder.html`.  

The code that provides the data to the template is located under `server/email/methods.js`. 

The fix is to ensure the code is properly referencing the logo image.  Hope that steers you in the right direction!

**Comment by lcrossan** [comment]
Hey, looks like I'm not allowed to push. 

```
$ git push origin issue-206

remote: Permission to codebuddiesdotorg/cb-v2-scratch.git denied to lcrossan.
fatal: unable to access 'https://github.com/codebuddiesdotorg/cb-v2-scratch/': The requested URL returned error: 403
```

Is this a config error on my end?

**Comment by sergeant-q** [comment]
@lpatmo do we have to add him as a collaborator to the repo? I can't recall if this is the case or if he is dealing with a local config issue.

**Comment by lpatmo** [comment]
Sorry, just saw this. @lcrossan And I'll add you as a collaborator so you can push the branch. And then please create a pull request for your branch. :) Thank you!

General note/talking to myself: in the future, let's start encouraging new contributors to fork this project instead of cloning directly so that they don't have to wait to be added in order to `git push` to their cloned repo. Since everyone is making pull requests anyway, this'll help solve some of the friction of contributing.

**Comment by lcrossan** [comment]
PR submitted. Thanks y'all!

**Comment by distalx** [comment]
resolved in #241 .
